From 93cfff1cd5ca8f418709e4fa0e1b1c9d5b15ddcf Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Fri, 12 Jun 2015 07:51:50 +0200
Subject: [PATCH 1014/1015] event: disable report_RHTSupport event and change
 URL for bugzilla

Set the URL for bugzilla to non-exist one because we do not want the centos
users report to bugzilla.

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/plugins/bugzilla.conf         | 2 +-
 src/plugins/rhtsupport_event.conf | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/plugins/bugzilla.conf b/src/plugins/bugzilla.conf
index 51648de..159b188 100644
--- a/src/plugins/bugzilla.conf
+++ b/src/plugins/bugzilla.conf
@@ -1,5 +1,5 @@
 # Bugzilla URL
-BugzillaURL = https://bugzilla.redhat.com/
+BugzillaURL = https://bugzilla.example.com/
 # yes means that ssl certificates will be checked
 SSLVerify = yes
 # your login has to exist, if you don have any, please create one
diff --git a/src/plugins/rhtsupport_event.conf b/src/plugins/rhtsupport_event.conf
index f6a9d47..4ccf9f5 100644
--- a/src/plugins/rhtsupport_event.conf
+++ b/src/plugins/rhtsupport_event.conf
@@ -1,3 +1,3 @@
-EVENT=report_RHTSupport
+#EVENT=report_RHTSupport
     # Submit an uReport and create a case in Red Hat Customer Portal
-    reporter-rhtsupport -u
+    # reporter-rhtsupport -u
-- 
1.8.3.1

